.travis.yml

```diff
@@ -91,160 +53,150 @@ matrix:
     fast_finish: true
 
     include:
-        # Try MacOS X.
+
+      # Try MacOS X.
         - os: osx
           stage: Cron tests
-          env: SETUP_CMD='test --remote-data=astropy'
-               CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang"
-               PIP_DEPENDENCIES="scikit-image jplephem $ASDF_PIP_DEP"
+          env: PYTHON_VERSION=3.7
+               TOXENV="test-alldeps"
+               TOXPOSARGS="--remote-data=astropy"
+               CONDA_DEPENDENCIES="clang"
                CCOMPILER=clang
+               USE_CI_HELPERS=True
```

Comment:
I was wondering about the status of ci-helpers with APE17. For what ci-helpers is still useful ?

Reply:
It could be for conda installation, as conda is defined just above, but I don't see tox-conda used in the `tox.ini`. Is it planned ?

Reply:
see https://github.com/astropy/ci-helpers/pull/429

Reply:
basically we would keep it to be able to centrally limit versions if necessary (let's say we know that a certain pytest, etc is incompatible)

Reply:
In this case it is still useful to set up different Python versions on non-Linux builds on Travis. We wouldn't need it if we switched to e.g. Azure for example, but that is beyond the scope of this PR.

Reply:
The issue with using tox-conda is that we would need https://github.com/tox-dev/tox-conda/issues/36 to be implemented properly to be able to use it. So instead here we never use tox-conda but do use conda on Travis to at least set up the Python environment. To simplify things I've now removed ``USE_CI_HELPERS`` and just decide whether to install conda based on the OS.